packages/ui/components/Chart.vue

```diff
@@ -3,6 +3,12 @@
     <div v-show="tooltip" class="tooltip">
       <span class="value">{{ formatNumber(value) }}</span>
       <span class="date"> {{ date }}</span>
+      <span class="date"> Deviation {{ deviation }}%</span>
+      <Heartbeat
+        class="countdown"
+        :milliseconds="86400000"
```

Comment:
Is this a common constant for all price feeds, or would it be adjusted depending on the actual configured value for each one of them ?